deployment/terraform/destroy.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package terraform
+
+import "fmt"
+
+// Destroy destroys the created load-test environment.
+func (t *Terraform) Destroy() error {
+	err := t.preFlightCheck()
+	if err != nil {
+		return err
+	}
```

Comment:
Is there any reason to not usiing `if err:= t.preFlightCheck(); ...`? Also we can directly return `t.RunCommand` below if we do this.

Reply:
No reason. It's just what was originally. I think we can return runCommand irrespective of that. Although it's a bit out of scope for this PR.

Reply:
doen.